src/core/readyState.js

```diff
@@ -20,16 +20,19 @@ function waitForDocumentReadyState () {
     emitReady();
   });
 }
-module.exports.waitForDocumentReadyState = waitForDocumentReadyState;
 
 /**
  * Signals A-Frame that everything is ready to initialize.
  */
-function emitReady () {
-  if (module.exports.canInitializeElements) { return; }
-  module.exports.canInitializeElements = true;
+export function emitReady () {
+  if (canInitializeElements) { return; }
+  canInitializeElements = true;
   setTimeout(function () {
     document.dispatchEvent(new CustomEvent('aframeready'));
   });
 }
-module.exports.emitReady = emitReady;
+
+/** Resets the canInitializeElements flag, used for testing */
+export function reset () {
+  canInitializeElements = false;
+}
```

Comment:
Anything imported through `import` is not mutable. The `readyState.test.js` used to override this value to simulate various scenarios. As a workaround this method is introduced.

An alternative would be to create a holder and mutate its contents, e.g.:
```js
export var canInitializeElements = {
  value: false
}
```

But I'd rather add this method to accommodate tests than to change all consumers because of the tests.